strictdoc/backend/reqif/p01_sdoc/sdoc_to_reqif_converter.py

```diff
@@ -578,22 +624,42 @@ def _convert_document_grammar_to_spec_types(
         ), "Only one grammar element is currently supported."
 
         for element in grammar.elements:
+            fields_names = element.get_field_titles()
+            statement_field_idx = fields_names.index("STATEMENT")
```

Comment:
This needs to be handled carefully. STATEMENT is a must but could be edge cases.